test/tangy-form-item_test.html

```diff
@@ -127,6 +167,23 @@
           element.querySelector('tangy-form-item').shadowRoot.querySelector('tangy-checkboxes').dispatchEvent(new CustomEvent('next'))
           assert.equal(element.querySelector('tangy-form-item').open, false)
         })
+        test('should autostop after 2 answers.', () => {
+          const sleep = (milliseconds) => new Promise((res) => setTimeout(() => res(true), milliseconds))
+          const element = fixture('TangyFormItemAutoStopFixture');
+          element.newResponse()
+          let formItem = element.querySelector('tangy-form-item')
+          let el1 = formItem.shadowRoot.querySelector('[name=fruit_selection1]').value
+                  .find(input => (input.name === 'Tangerine'))
+          el1.value = 'on'
+          let helpers = new TangyFormItemHelpers(formItem)
+          let hideInputsUponThreshhold = (input) => helpers.hideInputsUponThreshhold(input)
+          let disable = hideInputsUponThreshhold(formItem)
+          let el2 = formItem.shadowRoot.querySelector('[name=fruit_selection2]').value
+                  .find(input => (input.name === 'Tangerine'))
+          el2.value = 'on'
+          disable = hideInputsUponThreshhold(formItem)
```

Comment:
In the app, would you typically do an `input.dispatchEvent(new CustomEvent('next'), bubbles:true)` event to go to the next form, or does it need a more complex skip logic? - Does it need to stay as an on-change event, or can we automate this in the validate func?

Reply:
The use-case is much simpler - it simply disables the remaining questions once the threshhold has been reached.

![Kapture 2019-07-08 at 19 50 40](https://user-images.githubusercontent.com/861535/60831402-c3f2c600-a1b9-11e9-8080-30e08cd8b4ff.gif)